src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/MobileActionSheet.tsx

```diff
@@ -22,7 +22,7 @@ export const MobileActionSheet: SFC<MobileActionSheet> = ({
           : ""
 
         return (
-          <Container>
+          <Container mt={6}>
```

Comment:
Fixes issue where the mobile actions sheet was behind the nav. At the present moment, rather than taking over the whole screen, we need to move below the top nav. Will follow up on an appropriate fix later once we consolidate the ArtworkFilters. 

![mobile-sheet](https://user-images.githubusercontent.com/236943/58924053-16dae880-86f7-11e9-96dd-533793e387d3.gif)